content/riak/kv/2.1.4/using/cluster-operations/update-search-schema-reindex-data.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Updating & Reindexing Search Schemas"
+description: ""
+project: "riak_kv"
+project_version: "2.1.4"
+menu:
+  riak_kv-2.1.4:
+    name: "Updating / Reindexing Search Schemas"
+    identifier: "cluster_operations_search_schemas"
+    weight: 117
+    parent: "managing_cluster_operations"
+toc: true
+canonical_link: "https://docs.basho.com/riak/kv/latest/using/cluster-operations/update-reindex-search-schema"
+---
+
+The following page provides instruction on updating [search schemas](../../../developing/usage/search-schemas) as well as reindexing existing data.
+
+## Update the Search Schema
+
+First, make updates to the schema file, which is typically found here `$RIAK_HOST/search/index/famous`.
+
+For example, the following creates or updates an index called `famous` that uses the default schema:
+
+```bash
+curl -XPUT $RIAK_HOST/search/index/famous \ -H 'Content-Type: application/json' \ -d '{"schema":"_yz_default"}'
+```
+
+Then update the new schema within Riak KV by either:
+
+* Performing a [rolling restart](../../repair-recovery/rolling-restart) to trigger a Solr reload of the schema.
+* Or starting a `riak attach` session and run:
+  `yz_index:reload(<<"index_name">>).`
+
+> **Note:**
+>
+> Please note the `riak attach` method used above is a blocking, cluster-wide call but it will trigger a live reload of the Solr schema. New data will now contain the updated fields but existing data will not have their indexes updated.
+
+## Reindex Existing Data
+
+The first option to reindex existing data is rePUT all of your data.  You can do this by using a script or the [data migrator tool](https://github.com/dankerrigan/riak-data-migrator).
+
+The second option for reindexing existing data is to:
+
+1. Delete all of your Solr data from the index that you changed the schema on.  You do not need to delete the configurations.
+2. Expire the YZ AAE trees by entering a `riak attach` session and running the following command:  `rpc:multicall([node() | nodes()], yz_entropy_mgr, expire_trees, []).`
+3. Allow the trees to rebuild with the new index.  You can monitor the progress of this activity with `riak-admin search aae-status`
+
+## AAE Tuning
+
+By default search reindexing through AAE happens at the same rate as KV AAE. AAE trees are rebuilt at a rate of 1 tree per node per hour, and AAE trees are expired once every week. With these settings it likely will take a great deal of time for the index to rebuild after the trees expire.
+
+The [Active Anti Entropy][../active-anti-entropy] settings are described in our documentation. Of particular interest to you I think will be the following settings in `riak.conf` format, shown with their default values:
```

Comment:
It's weird that AAE is spelled out here but not above when it appears for the first time. It should probably get called out above and (abbreviated).

"Of particular interest will be the following `riak.conf`settings (shown with default values):"